content/en/user-guide/tools/lambda-tools/hot-reloading/index.md

```diff
@@ -1,50 +1,68 @@
-This way, any saved change inside your source file directly affects the already deployed Lambda function -- without any redeployment!
+LocalStack enables fast feedback cycles during development by automatically reloading your function code.
+Pro users can also hot-reload Lambda layers.
 
-## Covered Topics
-
-[Using the new Lambda provider](#using-the-new-lambda-provider):
-[Application Configuration Examples](#application-configuration-examples):
-* [Code hot-swapping for JVM Lambdas](#code-hot-swapping-for-jvm-lambdas)
-* [Code hot-swapping for Python Lambdas](#code-hot-swapping-for-python-lambdas)
-* Debugging Nodejs lambdas (under development)
+{{< alert title="Note" >}}
+**The magic S3 bucket name changed from `__local__` to `hot-reload` in Localstack&nbsp;v2.0**
 
-[Deployment Configuration Examples](#deployment-configuration-examples):
-* [Serverless Framework Configuration](#serverless-framework-configuration)
-* [AWS Cloud Development Kit (CDK) Configuration](#aws-cloud-development-kit-cdk-configuration)
-* [Terraform Configuration](#terraform-configuration)
+Please change your deployment configuration accordingly because the old value is an invalid bucket name.
+The configuration `BUCKET_MARKER_LOCAL` is still supported.
 
-[Useful Links](#useful-links)
+More information about the new lambda provider is available under [Lambda Provider Behavioral Changes]({{< ref "lambda-provider-v2" >}}).
+{{< /alert >}}
 
-## Using the new Lambda provider
-When using the new Lambda provider, hot-reloading does not conflict with fast execution times, and it is not needed to spawn a new container for every invocation.
-Also, hot reloading for Lambda layers (Pro) is now supported, it works identical to functions, but only 1 layer is allowed per function if hot-reloading is active in the layer.
+## Covered Topics
 
-For more information about behavioral changes, please consult the [Lambda Behavioral Changes]({{< ref "references/lambda-v2-provider" >}}) page.
+* [Hot Reloading Behavior](#hot-reloading-behavior)
+* [Application Configuration Examples](#application-configuration-examples):
+  * [Hot reloading for JVM Lambdas](#hot-reloading-for-jvm-lambdas)
+  * [Hot reloading for Python Lambdas](#hot-reloading-for-python-lambdas)
+  * Debugging Nodejs lambdas (under development)
+* [Deployment Configuration Examples](#deployment-configuration-examples):
+  * [Serverless Framework Configuration](#serverless-framework-configuration)
+  * [AWS Cloud Development Kit (CDK) Configuration](#aws-cloud-development-kit-cdk-configuration)
+  * [Terraform Configuration](#terraform-configuration)
+* [Useful Links](#useful-links)
+
+## Hot Reloading Behavior
+
+**Delay in code change detection:**
+It can take up to 700ms to detect code changes.
+In the meantime, invocations still execute the former code.
+Hot reloading triggers after 500ms without changes, can take up to 200ms until the change is notified.
```

Comment:
```suggestion
Hot reloading triggers after 500ms without changes, and it can take up to an additional 200ms until the reloaded code is in effect.
```